pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libusb }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.3";
```

Comment:
https://github.com/medusalix/xow/releases/tag/v0.4 has been released in the meanwhile.